app/views/users/edit.html.slim

```diff
@@ -1,4 +1,9 @@
-h2 = t('headings.users.edit', name: @user.email)
+h2 = t('headings.users.edit')
+
+ul.no-bullets
+  li = t('user.email_html', attr: @user.email)
```

Comment:
yeah, i combined with another PR. 

I didn't like how the edit user page looked, I wanted to give the page more context.